README.md

```diff
@@ -1,15 +1,16 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Bookstore Cms App
 
-# Project Name
+> This is a project that will help you Organize your books
 
-> Description the project.
+# Live Link 
+- https://dainty-treacle-081482.netlify.app/
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- React
+- Vs code
 
 ## Live Demo (if available)
```

Comment:
Great job @NatanimA 🤝

- [ ] [Optional] Kindly remove this Live Demo link since it is non-functional. You may maintain the link that is functioning in order to make your readme file more professional.